StreamChatUITestsAppUITests/MockServer/ChannelResponses.swift

```diff
@@ -15,6 +15,23 @@ extension StreamMockServer {
         server.register(MockEndpoint.channels) { [weak self] request in
             self?.updateChannelList(request)
         }
+        server.register(MockEndpoint.channel) { [weak self] request in
+            let json = TestData.toJson(request.body)
+
+            // Add members
+            var ids = json["add_members"] as? [String]
+            if let ids = ids {
+                return self?.addMembersToChannel(request, ids: ids)
+            }
+
+            // Remove members
+            ids = json["remove_members"] as? [String]
+            if let ids = ids {
+                return self?.removeMembersFromChannel(request, ids: ids)
+            }
+
+            return .badRequest(nil)
```

Comment:
Can we somehow hide this in a separate method, please?

Reply:
done